content/en/docs/reference/command-line-tools-reference/federation-apiserver.md

```diff
@@ -109,8 +109,8 @@ ServiceProxyAllowExternalIPs=true|false (DEPRECATED - default=false)
 StreamingProxyRedirects=true|false (BETA - default=true)
 SupportIPVSProxyMode=true|false (BETA - default=false)
 SupportPodPidsLimit=true|false (ALPHA - default=false)
-TaintBasedEvictions=true|false (ALPHA - default=false)
-TaintNodesByCondition=true|false (ALPHA - default=false)
+TaintBasedEvictions=true|false (BETA - default=true)
```

Comment:
federation api server?

Reply:
I just searched all occurrences of `TaintBasedEvictions`. IMO enabling a featuregate in `kube_features.go` is populated to all components, right?

Reply:
should be ; just not sure how federation handle such case: some feature upgrade from alpha to beta, does federation test them? maybe you can ping multicluster wg/sig .